.github/workflows/tests-unit-js.yml

```diff
@@ -2,10 +2,24 @@ name: JavaScript Unit Tests
 
 on:
   push:
+    # Only run if JS-related files changed.
+    paths:
+      - '**.js'
+      - 'package.json'
+      - 'package-lock.json'
```

Comment:
Do we need 
` - 'packages/**/package.json' `?

Reply:
I'll just need to replace `package.json` with `**/package.json`. ([as per glob tool](https://www.digitalocean.com/community/tools/glob?comments=true&glob=%2A%2A%2Fpackage.json&matches=false&tests=packages%2Fi18n%2Fpackage.json&tests=package.json))

Reply:
Yeah, just wonder.. I guess not.